subworkflows/local/profiling.nf

```diff
@@ -75,16 +75,20 @@ workflow PROFILING {
                     // as we don't run run on a per-sample basis due to huge datbaases
                     // so all samples are in one run and so sample-specific metadata
                     // unnecessary. Set as database name to prevent `null` job ID and prefix.
-                    def temp_meta = [ id: meta.db_name ]
+                    def temp_meta = [ id: db_meta.db_name ]
 
                     // Extend database parameters to specify whether to save alignments or not
-                    def new_db_meta = db_meta.clone()
                     def sam_format = params.malt_save_reads ? ' --alignments ./ -za false' : ""
+
+                    def db_meta_keys = db_meta.keySet()
+                    def new_db_meta = db_meta.subMap(db_meta_keys)
```

Comment:
```suggestion
                    def new_db_meta = db_meta.subMap(db_meta.keySet())
```
... or is there a reason we're being verbose about it? (same for the others of course)

Reply:
Uhh good point, I think I got overzealous with it out of paranoia. Let me refactor...